data-collection/epros-licensed-questionnaires/joint-surgery/oxford-hip-score.md

```diff
@@ -0,0 +1,25 @@
+
+---
+sidebar_position: 7
+title: Oxford Hip Score
```

Comment:
1. "title:" shows on the website. 
2. Section shows as "oxford-hip-score"

Please review and fix